dip-0003.md

```diff
@@ -232,8 +267,13 @@ Reorganisations of the chain must also correctly undo changes to the chain-tips
 
 A ProRegTx is invalid if any of these conditions are true:
 
+In the case of type 0:
   1. collateralOutpoint `hash` is null but an output with 1000 DASH is not present at position `n` of the ProRegTx outputs
   2. collateralOutpoint `hash` is not null but an output with 1000 DASH can't be found in the UTXO specified by the `hash` and `n`
+In the case of type 1:
```

Comment:
I think you need a paragraph break. This list doesn't render as desired (see https://github.com/dashpay/dips/blob/bls_dips_adaptations/dip-0003.md#validation-rules)
```suggestion

In the case of type 1:
```